README.md

```diff
@@ -21,6 +21,7 @@ Then you need to
 - Clone the repository you just created and named `${instanceName}-declarations`
 - Launch `./init.sh` and the script will guide you for the configuration
 - Commit the files to git `git add . && git commit -m "Initiate instance"`
+- Setup Dependabot through the `.github/dependabot.yml` configuration file
```

Comment:
This should be _before_ committing files 😉 
What exactly has to be “set up”?
Can't we just replace the maintainer team name automatically, like we do for the rest of the files? 🙂